packages/odyssey-react-mui/src/useContrastMode.tsx

```diff
@@ -0,0 +1,142 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  createContext,
+  useContext,
+  useRef,
+  useLayoutEffect,
+  useState,
+  useCallback,
+} from "react";
+import * as Tokens from "@okta/odyssey-design-tokens";
+
+export type ContrastMode = "lowContrast" | "highContrast";
+
+export type ContrastModeContextType = {
+  contrastMode: ContrastMode;
+};
+
+export const ContrastModeContext = createContext<ContrastModeContextType>({
+  contrastMode: "lowContrast",
+});
+
+export const useContrastModeContext = () => useContext(ContrastModeContext);
+
+const hexToRgb = (hex: string): string => {
+  const bigint = parseInt(hex.slice(1), 16);
+  const r = (bigint >> 16) & 255;
+  const g = (bigint >> 8) & 255;
+  const b = bigint & 255;
+  return `rgb(${r}, ${g}, ${b})`;
+};
+
+export const hueNeutral50Rgb = hexToRgb(Tokens.HueNeutral50);
+
+const isTransparentColor = (color: string): boolean =>
+  color === "rgba(0, 0, 0, 0)" || color === "transparent";
+
+const normalizeRgbaToRgb = (rgba: string): string =>
+  rgba.replace(/rgba\((\d+), (\d+), (\d+), \d+\)/, "rgb($1, $2, $3)");
+
+const getElementComputedBackgroundColor = (element: HTMLElement): string =>
+  window.getComputedStyle(element).backgroundColor;
```

Comment:
```suggestion
  window.getComputedStyle(element).getPropertyValue("backgroundColor");
```

Reply:
this causes the tests to fail and I think we'd have to refactor most of the tests to support it.